source/components/JsxParser.js

```diff
@@ -78,7 +79,7 @@ export default class JsxParser extends Component {
         })
         return object
       case 'Identifier':
-        return this.props.bindings[expression.name] || undefined
+        return this.bindings[expression.name] || undefined
```

Comment:
This seems cleaner as:
```javascript
return (props.bindings || {})[expression.name] || undefined
```

To make it more terse, the `this.bindings` line, above, could simply be `const bindings = props.bindings || {}`, and then this line could be `return bindings[expression.name] || undefined`

Reply:
Would you also please add one or more unit tests, to exercise this functionality and demonstrate that it works as-desired?